content/en/docs/sessions.md

```diff
@@ -1,13 +1,14 @@
 ---
 title: "Using Sessions to Configure Service Clients in the AWS SDK for Go"
 linkTitle: "Using Sessions"
+date: "2020-11-12"
 description: "Use sessions to define configurations for service clients."
 ---
```

Comment:
This file probably should be dropped, or renamed since there are no more `Session` type.

Reply:
Well this content if applicable remains to be migrated.